### DISCUSSION TITLE
Turn _clean_text optional

### DISCUSSION THREAD

**Discussion by LPerdu** [cdpierse/transformers-interpret#102] [Ideas]
Hello,

I believe turning the text cleaning step optional would ease results comparison.

Let's say you explain some sampled predictions from a production environment, you do not want the input text to be modified to allow apples to apples comparison.

``` Python
def _clean_text(self, text: str) -> str:
        text = re.sub("([.,!?()])", r" \1 ", text)
        text = re.sub("\s{2,}", " ", text)
        return text
```

Thanks !

**Comment by cdpierse** [comment]
Hi @LPerdu I think this would be a good addition, would you be interested in potentially looking to add this as a PR ?
